pkg/cmd/get/get_preview.go

```diff
@@ -92,7 +91,8 @@ func (o *GetPreviewOptions) CurrentPreviewUrl() error {
 	}
 	for _, env := range envList.Items {
 		if env.Spec.Kind == v1.EnvironmentKindTypePreview && env.Name == name {
-			log.Logger().Infof("%s", env.Spec.PreviewGitSpec.ApplicationURL)
+			// lets log directly to stdout for easy capture of the URL from shell scripts
+			fmt.Println(env.Spec.PreviewGitSpec.ApplicationURL)
```

Comment:
+1, I think we need to differentiate between logging and actual output of the command. The logging framework should not be used for the latter because it logs per default to stderr and because depending on the log level the expected output might not appear at all.

IMO, even better would be to let `CurrentPreviewUrl` return the URL as a string and handle the displaying in `Run`.